src/webview/components/kdbDataSourceView.ts

```diff
@@ -230,7 +231,12 @@ export class KdbDataSourceView extends LitElement {
               >${value}</vscode-option
             >
           `;
-        });
+        });  
+        */
+      this.selectedApi = "getData";
```

Comment:
Hardcoded should not be a solution

Reply:
This is an interim solution that is why I did not modify the code.

Reply:
Reverted to the previous solution instead of hardcoding.